diff --git a/lib/index.js b/lib/index.js
index 75fae74961306a992cde304e7311812c6e712de8..13d75bf8420abbbbd30ef365b3f08ea77b86f4a8 100644
--- a/lib/index.js
+++ b/lib/index.js
@@ -500 +500,4 @@ function taskkillProcessTree(pid) {
-	], { stdio: "ignore" });
+	], {
+		stdio: "ignore",
+		windowsHide: true
+	});
@@ -556 +559,2 @@ function spawnSubprocess(spec, internals = {}) {
-		detached: platform !== "win32"
+		detached: platform !== "win32",
+		windowsHide: platform === "win32"
